### PR TITLE
Reenable CMVS + bugfix in patch file output

### PR DESCRIPTION
**Pull request by mfehr**

### CODE REVIEW

**Comment by mfehr** [comment]
@simonlynen did you disable CMVS because of a problem with the build or because we haven't used it so far?

File: pmvs_catkin/src/pmvs/patchOrganizerS.cc [hunk 1]
```diff
@@ -93,47 +93,54 @@ void CpatchOrganizerS::writePatches2(const std::string prefix, bool bExportPLY,
...
     ofstr << "PATCHES" << endl
-          << (int)m_ppatches.size() << endl;
-    for (int p = 0; p < (int)m_ppatches.size(); ++p) {
-      Cpatch patch = *m_ppatches[p];
-      index2image(patch);
+        << (int)m_ppatches.size() << endl;
 
-      Vec3i color;
+    vector<Ppatch>::const_iterator bpatch = m_ppatches.begin();
+    vector<Ppatch>::const_iterator bend = m_ppatches.end();
```
Comment (simonlynen) [inline pmvs_catkin/src/pmvs/patchOrganizerS.cc]: const

**Comment by simonlynen** [comment]
lgtm

**Comment by simonlynen** [comment]
I think because it was broken back then,

**Pull request merged**